doc/fai-guide.txt

```diff
@@ -1691,7 +1697,7 @@ stick by just writing the content of the ISO file to your USB stick
 (here the stick is _/dev/sdf_).
 
 ----
- faiserver# dd if=fai-cd.iso of=/dev/sdf bs=1M
+ faiserver# dd if=faiserver-cd.iso of=/dev/sdf bs=1M
```

Comment:
this change didn't make it. intentionally so?

Reply:
Yes, I've cleaned the iso names. You proposed faiserver, I think fai-cd fits better. That's why there's the conflict.

Reply:
I think the idea driving this change has changed and is outdated. IIRC, I once assumed, it would build a dedicated ISO image for installing a FAI-server. At the time it may have looked like it, for some reason I cannot see now. Possibly PEBCAK.